src/ja_romaji.qml

```diff
@@ -44,5 +77,10 @@ KeyboardLayout {
         BackspaceKey {}
     }
 
-    SpacebarRow { }
+    SpacebarRow {}
+
+    type: "custom"
```

Comment:
For Component.onCompleted, I once tested deleting but it caused error. So I thought it's needed.
But I haven't examined much for why and I agree with you as look inside KeyboardBase.qml, it seems ok with only setting `type` to stop autocaps. So I have to test it more...

For the value of `type`, I think you are right.